.codespellignorewords

```diff
@@ -0,0 +1 @@
+simular
```

Comment:
You need to manage your tools better. Don't add typos to a list of "ignored spelling errors"